### ISSUE TITLE
Support a `make install` target that does not overwrite existing game files

### ISSUE THREAD

**Issue by jonm** [jonm/SillyMUD#26]
In particular, the source code comes with some empty files (e.g. `players`) in the `lib/` directory, but once you have the game up and running, a subsequent `make install` will put the empty versions over the existing ones, which is not the desired behavior. Really for the length-zero files, we should either have the game attempt to create them empty if they don't already exist (in which case we don't have to install them at all) or we should have the install target do a `touch` instead of a `cp`.

**Comment by jonm** [comment]
Empty files that would get installed currently include:

```
$ find lib -type f -empty
lib/25102.messages
lib/3093.messages
lib/3094.messages
lib/3095.messages
lib/3096.messages
lib/3097.messages
lib/3098.messages
lib/3099.messages
lib/bugs
lib/ideas
lib/killfile
lib/mortal.board
lib/pcobjs.obj
lib/players
lib/rhyodin
lib/skexie.board
lib/typos
lib/wiz.board
lib/wizlist
```

**Comment by jonm** [comment]
Fixed via #27 and #28.
